src/pages/RestaurantPage/RestaurantContainer.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect } from 'react';
+
+import { useDispatch, useSelector } from 'react-redux';
+
+import RestaurantDetail from './RestaurantDetail';
+import ReviewForm from './ReviewForm';
+import ReviewList from './ReviewList';
+
+import {
+  loadRestaurant,
+  sendReview,
+  updateReview,
+  resetReviewInput,
+} from '../../actions';
+
+import { get } from '../../utils';
+
+export default function RestaurantContainer({ restaurantId }) {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(loadRestaurant({ restaurantId }));
+  }, []);
+
+  const restaurant = useSelector(get('restaurant'));
+  const review = useSelector(get('review'));
+  const accessToken = useSelector(get('accessToken'));
+
+  if (!restaurant) {
+    return (
+      <p>Loading...</p>
+    );
+  }
+
+  function handleChange(event) {
+    const { target: { name, value } } = event;
+    dispatch(updateReview(name, value));
+  }
+
+  function handleSubmit() {
+    dispatch(sendReview({ restaurantId }));
+    dispatch(resetReviewInput());
+  }
+
+  return (
+    <>
+      <RestaurantDetail restaurant={restaurant} />
+      {accessToken ? (
+        <ReviewForm
+          review={review}
+          onChange={handleChange}
+          onSubmit={handleSubmit}
+        />
+      ) : null}
```

Comment:
삼항연산자 대신 이렇게 사용할 수도 있겠네요~ 

다만 항상 이 경우엔 체크하는 값이 0이 되지 않는지 확인을 해줘야합니다. 화면에 0이 노출되지 않도록 말이죠 ㅎㅎ
```suggestion
      {accessToken && (
        <ReviewForm
          review={review}
          onChange={handleChange}
          onSubmit={handleSubmit}
        />
      )}
```

Reply:
보여주신 걸로 바꿨습니다(JS 코딩의 기술에서도 이 부분이 있어서 기억이 나네요!)

accessToken이 0일 경우에도 (그럴 일은 없지만..) 그려져야하는 경우는 밑에처럼 삼항연산자와 단락평가를 같이 해서 표현할 수 있겠네요!

```
accessToken && accessToken === 0 ?  (
      <ReviewForm
        review={review}
        onChange={handleChange}
        onSubmit={handleSubmit}
      />
    ) : null
```